tests/widget-inline.test.js

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from '@playwright/test';
+
+test('inline widget', async ({ page }) => {
+  await page.goto('widget/inline');
+
+  await page.fill('text="User Name"', 'demouser');
+  await page.fill('text=Password', 'j56eKtae*1');
+  await page.locator('button', { hasText: 'Sign In' }).click();
+
+  const fullName = page.locator('#fullName');
+  const email = page.locator('#email');
+
+  expect(await fullName.innerText()).toBe('Full name: Demo User');
+  expect(await email.innerText()).toBe('Email: demo@user.com');
+});
```

Comment:
New Playwright didn't like one file have two `test` blocks, so moved this into its own file.

Reply:
you probably need a `describe` block?

Reply:
So, you're saying that using `test.describe` will isolate the test blocks? Is this what you mean: https://playwright.dev/docs/writing-tests#using-test-hooks?